zerver/views/message_edit.py

```diff
@@ -93,6 +94,30 @@ def fill_edit_history_entries(
     return formatted_edit_history
 
 
+def is_resolve_topic_request(
+    user_profile: UserProfile,
+    message_id: PathOnly[NonNegativeInt],
+    new_topic_name: OptionalTopic,
```

Comment:
This is used for determining if the message edit request is a resolve topic request, and calling the corresponding resolve topic method in actions. Not sure if `is_resolve_topic_request` is a good name.